mathics/core/pattern.py

```diff
@@ -399,6 +399,12 @@ def get_match_count(self, vars_dict: Optional[dict] = None) -> Tuple[int, int]:
         """The number of matches"""
         return (1, 1)
 
+    @property
+    def short_name(self) -> str:
```

Comment:
Using short_name came up in working on TraceEvaluation. I am surprised we never needed short_name for patterns before.